flowise-embed/src/utils/messageUtils.ts

```diff
@@ -5,7 +5,7 @@ export const messageUtils = {
   UPLOAD_BUTTON_LABEL: 'Fazer upload de documentos',
   UPLOADING_LABEL: 'Fazendo upload do documento',
   NEXT_CHECKLIST_BUTTON_LABEL: 'Próximo checklist',
-
+  NEW_CHAT_BUTTON_LABEL: '+ NOVO CHAT',
```

Comment:
Fazendo a alteração sugerida acima, aqui pode ficar apenas 
```suggestion
  NEW_CHAT_BUTTON_LABEL: 'Novo chat',